src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,34 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(actor);
```

Comment:
Let's use JPA's `persist()` instead, since it is recommended abstraction method, same for other classes
```suggestion
            session.persist(actor);
```